idd/Energy+.idd.in

```diff
@@ -4502,6 +4503,20 @@ Schedule:Constant,
        \type real
        \default 0
 
+Schedule:File:Shading,
+       \min-fields 1
+       \memo A Schedule:File:Shading points to a text computer file that has
+       \memo hourly/sub-hourly sunlit fraction data for all exterior surfaces.
```

Comment:
It's not really for "all" exterior surfaces though, right?  In the design/documentation it says that if surfaces are missing, they are simply not shaded.

Reply:
@lgu1234 Sorry not sub-hourly... Not necessary for sunlit fraction. Modified the text.